### DISCUSSION TITLE
Azure don't want to verify my academic status when I want to renew the free Student subscription

### DISCUSSION THREAD

**Discussion by tatarmb4s** [community/community#49777] [GitHub Education]
Hi!

My GitHub student dev pack ended 2 days ago, and I wanted to renew it, because I'm still a student.

In GitHub, I can access the student offers, and benefits, but when I try to activate them with the following link (https://aka.ms/azure4students) and select the GitHub method, I get this error:

We could not confirm your status as a student. Click here to begin the academic verification process. This process may take up to a few days. Once verification is complete, please come back and sign in with GitHub to redeem Azure for Students.

There are some screenshots:

I can renew my subscrition, after th academic verification, but it dont want to complete
![Screenshot 2023-03-11 103946](https://user-images.githubusercontent.com/94311853/224478864-21c5ddb7-517d-4be5-9f25-12ad0954bd20.png)
Azure don't want to verify my academic status:
![Screenshot 2023-03-11 103812](https://user-images.githubusercontent.com/94311853/224478866-50022e41-42b7-446b-8dd6-ad29aee27202.png)
GitHub edication page:
![Screenshot 2023-03-11 105657](https://user-images.githubusercontent.com/94311853/224478867-b582ca4a-e688-42dc-afa7-0f7132e7f8b5.png)
GitHub education status:
![Screenshot 2023-03-11 105600](https://user-images.githubusercontent.com/94311853/224478869-17182973-a0a6-43f3-b78c-21aa95049c1e.png)

https://imgur.com/a/x9AisBm

How can I fix it?

**Comment by bbkx226** [comment]
It sounds like you are experiencing an issue with the verification process to renew your GitHub Student Developer Pack. It's possible that there was an error in the verification process, or that there is a delay in confirming your student status.

The error message you received suggests that you need to complete the academic verification process. This process typically involves providing documentation to prove that you are currently enrolled as a student at an accredited institution. You can click on the link provided to begin this process, and follow the instructions to submit the necessary documentation.

Once your academic verification is complete, you should be able to sign in with your GitHub account and redeem the Azure for Students benefit. If you continue to experience issues, you can reach out to GitHub Support for further assistance.

**Reply by tatarmb4s** [reply]
I tried it, and I can't re-verify my account status, because the GitHubs submit button is disabled, to submit my request.

**Comment by samus-aran** [comment]
Hi @tatarmb4s 👋 

We've adjusted your current student status which will allow you to re-verify yourself. Please try and re-apply. 

Let us know how you get on 👍

**Reply by tatarmb4s** [reply]
I successfully submitted the student form, and get the student role again, but Azure still does not work. I waited some days, before trying it again, after reenrolling in the student program.  Same screen again @samus-aran @bbkx226 
![Screenshot 2023-03-18 095815](https://user-images.githubusercontent.com/94311853/226095833-2efa4039-6062-431e-a51c-54425db5b90d.png)

**Reply by bbkx226** [reply]
Hmm if that's the case, I'm sorry that I don't have any other ways to solve it

**Comment by HarshilChangrani** [comment]
I am also having the same issue

**Comment by fdelu** [comment]
Did either of you ever manage to solve it? @HarshilChangrani @tatarmb4s

**Reply by prashantmore277** [reply]
@HarshilChangrani @tatarmb4s  @fdelu anyone solve this issue please let me know only 7 days remains to renew my subscriptoinp🙏🙏🙏

**Reply by fdelu** [reply]
I ended up contacting support. The process took way too long but they ended up granting it

**Reply by prashantmore277** [reply]
Can you tell me how much time it takes because if i didnt get it within 7 days newal will get cancelled, and thanks for replying 👍

**Reply by fdelu** [reply]
Took me about 20 days

**Comment by Ayush-Kumar-web** [comment]
i am facing the same issue

**Reply by tatarmb4s** [reply]
I didnt solved the issue. I had to send a support request to Microsoft, and they couldnt fix the isssue. 

In the end, Microsoft ended up sending me a student package code, which had to be activated on a different microsoft account.

**Reply by prashantmore277** [reply]
I ended up contacting support. The process took way too long but they ended up granting it. it took 8 days for me.

**Reply by Dxuian** [reply]
@tatarmb4s  do i need to send the support request from the same account github student dev pack is enabled from?? and how long did the process take ? also did you need enter credit card information ?

**Reply by lerlerchan** [reply]
@tatarmb4s  how to send support request? through gitHub? or azure?

**Reply by tatarmb4s** [reply]
@lerlerchan @Dxuian I sent the support request on MS-t site. It took me about 1 week with multiple calls and screen shares, but the waiting payed out. It works currently withut any problems.

**Reply by lerlerchan** [reply]
@tatarmb4s I send a request ticket through azure support. 1 week  the Micosoft PIC contacted our campus IT admin. Wihtin the same week. the issue is solved. Tried azure support ticket

**Comment by seponik** [comment]
I have the same problem

**Comment by tatarmb4s** [comment]
Hello! The final solutions in the end, was that I reached out to Microsoft support, and after cupple of calls and tries, they gave me an Azure Activation Code, that works since 1.5 years ago.

@SepoHacks

**Reply by Kingofpers** [reply]
Thank you for your information and help.

**Reply by Faizankhan113** [reply]
can you give the link from where to contact their support ?? i am facing the same issue right now
